ironfish-rust/src/keys/mod.rs

```diff
@@ -210,10 +209,10 @@ impl SaplingKey {
     /// Adapter to convert this key to a proof generation key for use in
     /// sapling functions
     pub fn sapling_proof_generation_key(&self) -> ProofGenerationKey {
```

Comment:
This is a breaking change right? I couldn't figure out a way to make the construction the same for a new type (instead of using an extension trait).

Reply:
It is a breaking change, yes